src/charm.py

```diff
@@ -162,7 +162,7 @@ def _get_saml_config(self) -> Dict[str, Any]:
             Dictionary with the SAML configuration settings..
         """
         saml_fingerprints = {
-            "https://login.ubuntu.com/+saml": "32:15:20:9F:A4:3C:8E:3E:8E:47:72:62:9A:86:8D:0E:E6:CF:45:D5"
+            "https://login.ubuntu.com/+saml": "32:15:20:9F:A4:3C:8E:3E:8E:47:72:62:9A:86:8D:0E:E6:CF:45:D5"  # pylint: disable=line-too-long
```

Comment:
This can be broken into multiple lines, e.g.:

```Python
"https://login.ubuntu.com/+saml": (
    "32:15:20:9F:A4:3C:8E:3E:8E:47:"
    "72:62:9A:86:8D:0E:E6:CF:45:D5"
)
```

Reply:
Black puts it back on one line like  "32:15:20:9F:A4:3C:8E:3E:8E:47:" "72:62:9A:86:8D:0E:E6:CF:45:D5", but it is still less than the max-line-length, so it works. Thanks!